template/{% if documentation_tool %}docs{% endif %}/index.md.jinja

```diff
@@ -0,0 +1,24 @@
+# {{ project_name }}
+
+{{ project_description }}
+
+{%- if project_type == 'package' %}
+
+## Installation
+
+```bash
+pip install {{ project_name_kebab_case }}
+```
+
+## Quick Start
+
+```python
+import {{ project_name_snake_case }}
+
+# Your code here
+```
+
+## API Reference
+
+See the [API Reference](reference/index.md) for detailed documentation.
+{%- endif %}
```

Comment:
```suggestion
```

I would consider dropping all of this (they will need to edit it anyway) and providing only a minimal entrypoint for the user to write their own docs. Reason is that every change we make to this will trigger conflicts when they upgrade the template.